### DISCUSSION TITLE
Keyoxide

### DISCUSSION THREAD

**Discussion by rahilarious** [JulianPrieber/littlelink-custom#725] [Predefined Links]
https://keyoxide.org

Proposal to add predefined link with Keyoxide logo and buttons in Linkstack. 

Adding linkstack to Keyoxide would be cool too

**Comment by MagicLike** [comment]
https://community.keyoxide.org/d/77-add-littlelink-custom

**Reply by JulianPrieber** [reply]
![image](https://github.com/LinkStackOrg/LinkStack/assets/60265788/5f581429-5156-4ebe-9c83-67adfab38b9b)
That's fairly easy to do

**Reply by JulianPrieber** [reply]
What data do they need?

**Reply by MagicLike** [reply]
I don't really know, I just made the suggestion some while ago O_o

**Comment by rahilarious** [comment]
Just to be clear I'm asking to add predefined link with Keyoxide logo and buttons in Linkstack. 

Not adding linkstack to Keyoxide :) Both of them would be cool though.

**Reply by JulianPrieber** [reply]
My understanding is that the two go hand in hand

**Reply by beedell-roke** [reply]
> https://github.com/orgs/LinkStackOrg/discussions/725#discussioncomment-8507728

@rahilarious, you should add that to https://github.com/orgs/LinkStackOrg/discussions/725#discussion-6215597, then. Without it, it's not obvious what you want until the reader scrolls to https://github.com/orgs/LinkStackOrg/discussions/725#discussioncomment-8507728:~:text=just%20to%20be%20clear%20i'm%20asking%20to%20add%20predefined%20link%20with%20keyoxide%20logo%20and%20buttons%20in%20linkstack.
